udp/src/main/scala/akka/stream/alpakka/udp/impl/UdpBind.scala

```diff
@@ -31,7 +35,10 @@ import scala.concurrent.{Future, Promise}
 
   override def preStart(): Unit = {
     implicit val sender = getStageActor(processIncoming).ref
-    IO(Udp) ! Udp.Bind(sender, localAddress)
+    options match {
+      case Some(socketOptions) => IO(Udp) ! Udp.Bind(sender, localAddress, socketOptions)
+      case None => IO(Udp) ! Udp.Bind(sender, localAddress)
+    }
```

Comment:
I agree with https://github.com/akka/alpakka/pull/2660#discussion_r631845751 -- the wrapping into `Option` shouldn't be necessary. And I assume you may even pass an empty sequence of options to `Udp.Bind` to mean the same as no parameter.

Reply:
Apologies, I missed that updated comment when I did the last updates. Hopefully I got everything now.